plenum/server/view_change/pre_view_change_strategies.py

```diff
@@ -125,17 +125,16 @@ def unstash_messages(self):
 
     def _set_req_handlers(self):
         node_msg_router = self.node.nodeMsgRouter
-        replica_msg_router = self.replica.inBoxRouter
+        replicas_external_bus = self.replica._external_bus
 
         if ViewChangeStartMessage not in node_msg_router.routes:
             processor = partial(VCStartMsgStrategy.on_view_change_started,
                                 self.node)
             node_msg_router.add((ViewChangeStartMessage, processor))
 
-        if ViewChangeContinueMessage not in replica_msg_router.routes:
-            processor = partial(VCStartMsgStrategy.on_view_change_continued,
-                                self.replica)
-            replica_msg_router.add((ViewChangeContinueMessage, processor))
+        processor = partial(VCStartMsgStrategy.on_view_change_continued,
+                            self.replica)
+        replicas_external_bus.subscribe(ViewChangeContinueMessage, processor)
```

Comment:
Previously there was a check to avoid double subscription. Do we need to so similar thing here or that was just over-cautiousness?